src/Ocelot/DownstreamUrlCreator/Middleware/DownstreamUrlCreatorMiddleware.cs

```diff
@@ -125,22 +126,24 @@ private static void RemoveQueryStringParametersThatHaveBeenUsedInTemplate(Downst
             }
         }
 
-        private static string GetPath(DownstreamPath dsPath)
+        private static string GetPath(DownstreamPath dsPath)
         {
-            return dsPath.Value.Substring(0, dsPath.Value.IndexOf('?', StringComparison.Ordinal));
+            return dsPath.Value.Substring(0, dsPath.Value.IndexOf('?', StringComparison.Ordinal));
         }
 
-        private static string GetQueryString(DownstreamPath dsPath)
+        private static string GetQueryString(DownstreamPath dsPath)
         {
-            return dsPath.Value.Substring(dsPath.Value.IndexOf('?', StringComparison.Ordinal));
+            return dsPath.Value.Substring(dsPath.Value.IndexOf('?', StringComparison.Ordinal));
         }
 
-        private static bool ContainsQueryString(DownstreamPath dsPath)
+        private static bool ContainsQueryString(DownstreamPath dsPath)
         {
-            return dsPath.Value.Contains('?');
+            return dsPath.Value.Contains('?');
         }
 
-        private (string Path, string Query) CreateServiceFabricUri(DownstreamRequest downstreamRequest, DownstreamRoute downstreamRoute, List<PlaceholderNameAndValue> templatePlaceholderNameAndValues, Response<DownstreamPath> dsPath)
+        private (string Path, string Query) CreateServiceFabricUri(DownstreamRequest downstreamRequest,
+            DownstreamRoute downstreamRoute, List<PlaceholderNameAndValue> templatePlaceholderNameAndValues,
+            Response<DownstreamPath> dsPath)
```

Comment:
I see zero changes in method signature!
But, If you split parameters, I suggest to put each parameter onto new line 👇 
```suggestion
        private (string Path, string Query) CreateServiceFabricUri(
            DownstreamRequest downstreamRequest,
            DownstreamRoute downstreamRoute,
            List<PlaceholderNameAndValue> templatePlaceholderNameAndValues,
            Response<DownstreamPath> dsPath)
```

Reply:
Fixed in commit https://github.com/ThreeMammals/Ocelot/pull/1348/commits/3c7781e7ceccf0a54e1feae91a07c78e12bcfa3d ✔️ 
The version from the develop branch was utilized.